test/oranif_eunit.erl

```diff
@@ -696,6 +696,37 @@ stmtGetQueryInfoFail(#{session := Conn} = TestCtx) ->
     ),
     dpiCall(TestCtx, stmt_close, [Stmt, <<>>]).
 
+stmtGetInfo(#{session := Conn} = TestCtx) ->
+    Stmt = dpiCall(
+        TestCtx, conn_prepareStmt, 
+        [Conn, false, <<"select 1337 from dual">>, <<>>]
+    ),
+    #{
+        isDDL := IsDDL, isDML := IsDML,
+        isPLSQL := IsPLSQL, isQuery := IsQuery,
+        isReturning := IsReturning, statementType := StatementType
+    } = dpiCall(TestCtx, stmt_getInfo, [Stmt]),
+    ?assert(is_atom(IsDDL)),
+    ?assert(is_atom(IsDML)),
+    ?assert(is_atom(IsPLSQL)),
+    ?assert(is_atom(IsQuery)),
+    ?assert(is_atom(IsReturning)),
+    ?assert(is_atom(StatementType)),
+    dpiCall(TestCtx, stmt_close, [Stmt, <<>>]).
```

Comment:
all statement type (expect select) are not covered!
See: https://coveralls.io/builds/24530885/source?filename=c_src/dpiStmt_nif.c#L342